src/config/firebase.js

```diff
@@ -0,0 +1,20 @@
+import admin from 'firebase-admin';
```

Comment:
Add ESLint ignore to this file as import will fail on GitHub due to gitignore

Reply:
okay okay cool